cljam.rb

```diff
@@ -1,8 +1,8 @@
 class Cljam < Formula
   desc "Tools for manipulating DNA Sequence Alignment/Map (SAM)"
   homepage "https://chrovis.github.io/cljam/"
-  url "https://github.com/chrovis/cljam/releases/download/0.8.0/cljam", :using => :nounzip
-  sha256 "1090a80b24da12ddff1e003f0eae01c92bcddb38215d66618332ac19d36acf9f"
+  url "https://github.com/chrovis/cljam/releases/download/0.8.3/cljam", :using => :nounzip
+  sha256 "7bedabc234106cac532a52fe8fee722049312e17a7b62e6248b156eacc6d4712"
 
   depends_on :java
```

Comment:
```console
$ brew install cljam.rb
Error: cljam: Unsupported special dependency :java
```

`depends_on :java` wad deprecated, so please change it to `depends_on "openjdk"`.

Sorry, I pushed [a fix branch](https://github.com/xcoo/homebrew-formulae/tree/fix/cljam-depends-on-openjdk) one year ago, but I forgot to create a pull request.

Reply:
Thank you, I didn't notice that 🙇 I fixed it in 72bfa6f